visualizations/frontend/filters/TVisGrouper.vue

```diff
@@ -0,0 +1,43 @@
+<script>
+    export default {
+        props: {
+            tClass: {
```

Comment:
Yes it does, but due to the nature of this component, it is actually creating the template on the fly instead of having a predefined template, so any class we apply to it directly will not really work because there is no default template code to apply those classes.

Reply:
ah ok.